mk/compile.mk

```diff
@@ -95,6 +107,11 @@ comp-objcpy-cmd-$2 = $$(OBJCOPY$(sm)) \
 	--rename-section .rodata.str1.1=.rodata.str1.1.$1 \
 	$2
 
+# Assign defaults if unassigned
```

Comment:
It doesn't work in the else since that ifeq is only performed on C files, not on S files.